iam-policy-s3-buckets.tf

```diff
@@ -30,8 +30,8 @@ resource "aws_iam_role_policy" "s3_data_for_hms_readwrite" {
                               "s3:PutObjectVersionTagging"
                             ],
                   "Resource": [
-                                "${join("\",\"", formatlist("arn:aws:s3:::%s", local.apiary_data_buckets))}",
-                                "${join("\",\"", formatlist("arn:aws:s3:::%s/*", local.apiary_data_buckets))}"
+                                "${join("\",\"", formatlist("arn:aws:s3:::%s", local.apiary_iam_buckets))}",
```

Comment:
we probably shoud create another inline policy and just add s3-inventory read permissions to both readwrite and readonly metastore,  we shouldn't have need for users to write to inventory bucket.

Reply:
1. Deploy `apiary-data-lake` with S3 inventory enabled.
2. Terraform enables inventory for the apiary buckets, but nothing will run for up to 24 hours, so the inventory bucket is empty.
3. HMS docker image starts up and tries to create the tables.  When the Hive `create table` statement is run, hive will see that the table folder doesn't yet exist in S3 and will try to create the `<tableName>_$folder$` object in the inventory bucket. <-- *This will fail w/o write permissions in IAM for R/W HMS*

Is there a way around this while still keeping users from writing?

Reply:
Hacky, but could we pre-create that folder here? Hopefully there's a better way but...

Reply:
Metastore docker image now won't try to create the table until the data is written the first time by AWS, so scheduled job doesn't need write privs to bucket.